src/sql/workbench/services/contextualization/common/serverContextualizationService.ts

```diff
@@ -80,14 +81,27 @@ export class ServerContextualizationService extends Disposable implements IServe
 	 * @param ownerUri The URI of the connection to get context for.
 	 */
 	public async getServerContextualization(ownerUri: string): Promise<azdata.contextualization.GetServerContextualizationResult> {
+		const copilotExt = await this._extensionService.getExtension('github.copilot');
```

Comment:
Would suggest making a helper for this since you do it a couple times in various places